lib/linter.js

```diff
@@ -145,17 +145,15 @@ function parseJsonConfig(string, location) {
  */
 function parseListConfig(string) {
     const items = {};
+    const reg = /[\w-/]+/g;
```

Comment:
Should this hyphen be escaped?

Reply:
no need in `[]`

Reply:
oh, before `/` , it's no need. I was confused.